bgpd/bgp_updgrp.c

```diff
@@ -907,7 +907,8 @@ static void update_subgroup_remove_peer_internal(struct update_subgroup *subgrp,
 {
 	assert(subgrp && paf);
```

Comment:
This should probably be part of the assert(subgrp && paf && subgrp->update_group)